pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -54,14 +54,15 @@ load_user_files <- function(input, pipeline_config, prev_out, input_dir = "/inpu
 read_10x_files <- function(config, input_dir) {
   counts_list <- list()
   annot_list <- list()
+  features_types_list <- list()
 
   samples <- config$samples
 
-
   for (sample in samples) {
     sample_dir <- file.path(input_dir, sample)
     sample_fpaths <- list.files(sample_dir)
     annot_fpath <- file.path(sample_dir, "features.tsv.gz")
```

Comment:
1. if this `features.tsv.gz` is a constant I'd add it to the top of the file or something.
2. how does this work if the file is called `genes.tsv.gz` which we say we support in the UI?

Reply:
They are already renamed by this stage I believe, we would need to check the UI but I'm fairly certain that this was discussed when we originally wrote this

Reply:
they are renamed in the UI, and in S3 always stored as features.tsv.gz